### PR TITLE
[Status Dashboard] Implement graphical view for deployed siddhi application

### PR DESCRIPTION
**Pull request by GowthamyVaseekaran**
image

### CODE REVIEW

**Comment by erangatl** [comment]
Shall we change the icon border color to white as well.

File: components/org.wso2.carbon.status.dashboard.web/public/icons/aggregate.svg [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+ ~   Copyright (c) WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (pcnfernando) [inline components/org.wso2.carbon.status.dashboard.web/public/icons/aggregate.svg]: use correct wso2 license with year

File: components/org.wso2.carbon.status.dashboard.web/public/icons/function.svg [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--x
+ ~   Copyright (c) WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (pcnfernando) [inline components/org.wso2.carbon.status.dashboard.web/public/icons/function.svg]: correct license with year. check all places

File: components/org.wso2.carbon.status.dashboard.web/public/icons/streams.svg [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+ ~   Copyright 2017, WSO2 Inc. (http://www.wso2.com)
```
Comment (pcnfernando) [inline components/org.wso2.carbon.status.dashboard.web/public/icons/streams.svg]: correct license year

File: components/org.wso2.carbon.status.dashboard.web/public/icons/table.svg [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+ ~   Copyright 2017, WSO2 Inc. (http://www.wso2.com)
```
Comment (pcnfernando) [inline components/org.wso2.carbon.status.dashboard.web/public/icons/table.svg]: correct license year

File: components/org.wso2.carbon.stream.processor.core/src/main/java/org/wso2/carbon/stream/processor/core/impl/SiddhiAppsApiServiceImpl.java [hunk 5]
```diff
@@ -391,6 +410,329 @@ public Response siddhiAppsStatsEnable(boolean statsEnabled) throws NotFoundExcep
...
+        Map<String, WindowDefinition> windowDefinitionMap = siddhiAppRuntime.getWindowDefinitionMap();
+        Map<String, AggregationDefinition> aggregationDefinitionMap = siddhiApp.getAggregationDefinitionMap();
+        for (Map.Entry<String, StreamDefinition> entry : streamDefinitionMap.entrySet()) {
+            if (entry.getKey().equals(inputStream)) {
+                siddhiAppElements.setInputStreamSiddhiApp(String.valueOf(entry.getValue()));
+                siddhiAppElements.setInputStreamType(Constants.STREAM_TYPE);
+                break;
+            }
+        }
+
```
Comment (pcnfernando) [inline components/org.wso2.carbon.stream.processor.core/src/main/java/org/wso2/carbon/stream/processor/core/impl/SiddhiAppsApiServiceImpl.java]: remove unnecessary spaces

File: components/org.wso2.carbon.stream.processor.core/src/main/java/org/wso2/carbon/stream/processor/core/impl/SiddhiAppsApiServiceImpl.java [hunk 6]
```diff
@@ -391,6 +410,329 @@ public Response siddhiAppsStatsEnable(boolean statsEnabled) throws NotFoundExcep
...
+            for (Element element : annotation.getElements()) {
+                siddhiAppElements.setQueryName(element.getValue());
+            }
+        }
+        if (siddhiAppElements.getQueryName() == null) {
+            siddhiAppElements.setQueryName(Constants.DEFAULT_QUERY_NAME);
+        }
+    }
+
+
```
Comment (pcnfernando) [inline components/org.wso2.carbon.stream.processor.core/src/main/java/org/wso2/carbon/stream/processor/core/impl/SiddhiAppsApiServiceImpl.java]: Remove unecessary spaces. Check all places

File: components/org.wso2.carbon.stream.processor.core/src/main/java/org/wso2/carbon/stream/processor/core/impl/SiddhiAppsApiServiceImpl.java [hunk 7]
```diff
@@ -391,6 +410,329 @@ public Response siddhiAppsStatsEnable(boolean statsEnabled) throws NotFoundExcep
...
+        loadSources(siddhiAppRuntime, siddhiAppElements, inputStream, appData);
+        loadSinks(siddhiAppRuntime, siddhiAppElements, inputStream, appData);
+    }
+
+    /**
+     * Obtain the siddhi app and the type of output stream.
+     */
+    private void loadOutputData(SiddhiApp siddhiApp, SiddhiAppRuntime siddhiAppRuntime, String outputStream, String
+            appData, SiddhiAppElements siddhiAppElements) {
+        Map<String, StreamDefinition> streamDefinitionMap = (siddhiAppRuntime.getStreamDefinitionMap());
```
Comment (pcnfernando) [inline components/org.wso2.carbon.stream.processor.core/src/main/java/org/wso2/carbon/stream/processor/core/impl/SiddhiAppsApiServiceImpl.java]: we can remove the extra brackets

**Pull request merged**